tests/test_parser_stress.py

```diff
@@ -0,0 +1,350 @@
+from io import BytesIO
+from itertools import chain
+import math
+from numpy import random
+from unittest import TestCase
+
+from requests_toolbelt import MultipartEncoder
+
+from streaming_form_data import StreamingFormDataParser
+from streaming_form_data.targets import ValueTarget
+
+
+def get_random_bytes(size, seed):
+    random.seed(seed)
+    return random.bytes(size)
+
+
+def get_hyphens_crlfs(size, seed):
+    random.seed(seed)
+    return random.choice([b'\r', b'\n', b'-'], size,
+                         p=[0.25, 0.25, 0.5]).tobytes()
+
+
+def is_prime(n):
+    if n % 2 == 0 and n > 2:
+        return False
+    return all(n % i for i in range(3, int(math.sqrt(n)) + 1, 2))
+
+
+def is_power_of(x, base):
+    n = x
+    while(n > 0):
+        if n == 1:
+            return True
+        if n % base:
+            return False
+        n /= base
+    raise Exception('is_power_of: unexpected result with x = ' +
+                    str(x) + ' and base = ' + str(base))
+
+
+def is_square(n):
+    sq = int(math.sqrt(n))
+    return sq * sq == n
+
+
+def is_multiple(n, base):
+    return n % base == 0
+
+
+def is_interesting_number(n):
+    if n < 1:
+        return False
+
+    if n <= 1000 and is_prime(n):
+        return True
+
+    if is_power_of(n, 2) \
+            or is_power_of(n-1, 2) \
+            or is_power_of(n+1, 2):
+        return True
+
+    if is_power_of(n, 10) \
+            or is_power_of(n-1, 10) \
+            or is_power_of(n+1, 10):
+        return True
+
+    if is_multiple(n, 1024) \
+            or is_multiple(n-1, 1024) \
+            or is_multiple(n+1, 1024):
+        return True
+
+    if is_multiple(n, 1000) \
+            or is_multiple(n-1, 1000) \
+            or is_multiple(n+1, 1000):
+        return True
+
+    if is_square(n):
+        return True
+
+    if n <= 64:
+        return True
+
+    return False
+
+
+def is_interesting_number_short(n):
```

Comment:
Same here. Would be cool to know when a number is considered as interesting. A docstring (something like https://stackoverflow.com/a/3899154/179729) would be nice. :)